### DISCUSSION TITLE
Detected CSRF Attack! Do not do evil things with pimcore ... ;-)

### DISCUSSION THREAD

**Discussion by jonyking** [pimcore/pimcore#8275] [Q&A / Support]
Hello,

I've got the following error message, when I tried to login in the backend:

`Detected CSRF Attack! Do not do evil things with pimcore ... ;-)`

I've tried the following:

- Clean cache of browser
- Use a new browser in privacy mode
- Clear cache
- Reset Permissions
- Cache warming
- After new install of pimcore skeleton

Does anybody have some suggestions?

**Comment by folkraceshop** [comment]
I have the same problem. I manage to login using http but not with https ssl. The first login try just put me back to login page. Second try I get "
ERROR 08:55:40 | security | Detected CSRF attack on /admin/login/login{     "request": "/admin/login/login" }". I run NGINX, could it be some parameter in nginx.conf? I have got it from installation page.
